components/mpas-ocean/src/shared/mpas_ocn_frazil_forcing.F

```diff
@@ -448,6 +452,13 @@ subroutine ocn_frazil_forcing_build_arrays(domain, meshPool, forcingPool, stateP
       frazilSalinityTendency = 0.0_RKIND
       frazilLayerThicknessTendency = 0.0_RKIND
 
+      ! initialize frazil salinity conservation field
+      lowSalinityFrazilIce = 0.0_RKIND
+
+      ! Determine if using porosity to compute internal frazilSalinity
+      usePorosity = 0 ! use config_frazil_sea_ice_reference_salinity
+      if (config_use_frazil_ice_porosity) usePorosity = 1
+
```

Comment:
not a big deal, but couldn't this be moved to some init routine?  The flag isn't changing during the simulation.

Reply:
Good point.  I'll see what I can do.

Reply:
@vanroekel :  I've  moved the code to ocn_frazil_forcing_init.